File: app/src/main/java/br/com/douglas/fukuhara/bank/network/vo/UserError.java
package br.com.douglas.fukuhara.bank.network.vo;

import com.google.gson.annotations.SerializedName;

public class UserError {
    @SerializedName("code")
    int code;
    @SerializedName("message")
    String message;

    public int getCode() {
        return code;
    }

    public String getMessage() {
        return message;
    }
}

File: app/src/main/java/com/develop/zupp_bank/domain/models/UserError.kt
package com.develop.zupp_bank.domain.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class UserError (

    @SerializedName("code")
    @Expose val code: Int,

    @SerializedName("message")
    @Expose val message: String

)